src/yummly/logentries_timbre_appender.clj

```diff
@@ -39,25 +39,38 @@
                         ;; be converted to json
                         (dissoc :properties))
                    errors)
-             (catch Exception e
+             (catch Exception _
                (remove :omitted errors)))))))
 
+(def illegal-key-characters #"[^a-zA-Z0-9@\$\._]")
+
+(defn clean-key
+  "InsightOps only supports alphanumeric values along with `@`, `$`, `_`, and `.` in keys. This function
+  replaces all illegal characters in a string/keyword with a `.`. See https://docs.logentries.com/docs/json#section-kvp-parsing-specification for details."
```

Comment:
i think it would be more natural to replace `-` with `_`

Reply:
@vgeshel should I just replace all of the illegal characters with `_` then?